.github/workflows/ci.yml

```diff
@@ -18,14 +18,12 @@ jobs:
         with:
           submodules: true
       - uses: dtolnay/rust-toolchain@stable
-      - uses: Swatinem/rust-cache@dd05243424bd5c0e585e4b55eb2d7615cdd32f1f
-        with:
-          workspaces: starknet-foundry
+      - uses: Swatinem/rust-cache@v2
```

Comment:
I'd still use specific commit instead of just `@v2`, we can bump it to most recent.